apps/admin-x-activitypub/src/views/Notifications/Notifications.tsx

```
@@ -402,27 +298,27 @@ const Notifications: React.FC<NotificationsProps> = () => {
                                                 </div>
                                                 {(
-                                                    (group.type === ACTIVITY_TYPE.CREATE && group.object?.inReplyTo) ||
-                                                    (group.type === ACTIVITY_TYPE.LIKE && !group.object?.name && group.object?.content) ||
-                                                    (group.type === ACTIVITY_TYPE.REPOST && !group.object?.name && group.object?.content)
+                                                    (group.type === 'reply' && group.inReplyTo) ||
+                                                    (group.type === 'like' && !group.post?.name && group.post?.content) ||
+                                                    (group.type === 'repost' && !group.post?.name && group.post?.content)
                                                 ) && (
                                                     <div
-                                                        dangerouslySetInnerHTML={{__html: stripHtml(group.object?.content || '')}}
+                                                        dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '')}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)